content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,142 @@
+| Member roles                                  | • Admin<br />• Viewer<br />• Developer<br />• Collaborator       | Admin only                                               | • Admin<br />• Viewer<br />• Developer<br />• Collaborator                                                                                                                | • Admin<br />• Viewer<br />• Developer<br /> |
+| Environments                                  | 7                                                                | 2                                                        | Unlimited<br /><br /> <strong>Quota</strong><br /><br />5 <br /><br /><strong>Overage</strong><br /><br />Each additional member at a fixed price                         | Custom                                       |
+| **Data Proxy**<br /><br /> - CPU usage        | 100,000,000<br /><br />CPU&nbsp;ms<br /><br />(hard limit)       | 3,000,000<br /><br />CPU&nbsp;ms<br /><br />(hard limit) | Unlimited<br /><br /><strong>Quota</strong><br /><br />30,000,000 CPU&nbsp;ms<br /><br /><strong>Overage</strong><br /><br />Each additional CPU&nbsp;ms at a fixed price | Unlimited<br /><br />Custom pricing          |
+| **Data Proxy**<br /><br /> -&nbsp;Concurrency | • 3 QEIDs\* per project<br />• 5 QEIDs\* per account/team member | 100 QEIDs\* per project                                  | 100 QEIDs\* per project                                                                                                                                                   | Custom                                       |
+
+<Admonition type="warning">
+
+\*_**QEIDs**_ refers to the number of available query engines that the Prisma Data Proxy can start and keep alive to manage the connection pool to the database for a project.
+
+</Admonition>
+
+## Quotas
+
+### Quotas for team members and environments
+
+- You can add as many team members and environments as needed to a project.
+- We bill for each additional team member and environment that exceeds the quota at a fixed price based on the plan.
+- When you add a team member or environment that exceeds your quota, we bill you with a pro-rated charge until the end of the current month with the next month's invoice.
+- After that, we bill you monthly at the fixed price per collaborator or environment for your plan.
+- If at any point during the current month you remove any previously added collaborators or environments from the monthly quota, we issue a prorated refund for the unused time until the end of the month.
+
+### Quotas for the Prisma Data Proxy
+
+- Each plan includes different Data Proxy CPU usage quotas which reset at the beginning of every month.
+- The `Early Adopter` and `Free` plans have a hard limit of monthly Data Proxy CPU usage.
+- If a project reaches the hard limit on `Early Adopter` and `Free` plans, we will send you a notification in advance and you can consider upgrading to a paid plan.
```

Comment:
Suggestion, something like:

- We will send notifications in advance ahead of the hard limits, so you can consider upgrading to a paid plan to avoid service interruptions.
Member

Reply:
Resolving.